pkg/cdi/doc.go

```diff
@@ -99,40 +86,40 @@
 // CDI Spec cache is up to date before performing device injection.
 // A code snippet similar to the following accmplishes that:
 //
-//	import (
-//	    "fmt"
-//	    "strings"
+//		import (
```

Comment:
Similar here.

Reply:
yeah, it's my editor reformatted those. Should be fixed now. Unfortunately with more reformatting.